src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -55,75 +55,106 @@
+                                                                                                :death-rate
+                                                                                                :death-rate
+                                                                                                start-yr-avg-mort
+                                                                                                end-yr-avg-mort)]
+                                                               {:initial-projected-mortality-rates
+                                                                (cf/apply-national-trend projected-rates-jumpoffyr
+                                                                                         future-mortality-trend-assumption
+                                                                                         first-proj-yr
+                                                                                         last-proj-yr
+                                                                                         mort-scenario
+                                                                                         :death-rate)})
+                                 :applynationaltrend-trend (let [projected-rates-jumpoffyr
+                                                                 (cf/jumpoffyr-method-trend historic-asmr
+                                                                                            :death-rate
+                                                                                            :death-rate
+                                                                                            start-yr-avg-mort
+                                                                                            end-yr-avg-mort)]
+                                                             {:initial-projected-mortality-rates
+                                                              (cf/apply-national-trend projected-rates-jumpoffyr
+                                                                                       future-mortality-trend-assumption
+                                                                                       first-proj-yr
+                                                                                       last-proj-yr
+                                                                                       mort-scenario
+                                                                                       :death-rate)}))]
+    asmr-projection-method))
 
-(defworkflowfn project-deaths-from-fixed-rates
-  "Takes a dataset with population at risk from the current year of the projection
-  loop and another dataset with fixed death rates for the population. Returns a
-  dataset with a column of deaths, which are the product of popn at risk & death rates"
-  {:witan/name :ccm-mort/project-deaths-fixed-rates
+(defworkflowfn project-asmr-1-0-0
```

Comment:
so if I wanted a workflow function with the trend-fixed variant, for example, would you suggest we add a new workflow function version 1.0.1?

I'm wondering whether to keep project-asmr-average-fixed as its a bit more informative than 1-0-0